packages/paste-libraries/data-visualization/src/usePasteHighchartsTheme.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+import {ThemeContext} from '@twilio-paste/styling-library';
+import type {ThemeShape} from '@twilio-paste/theme';
+import type Highcharts from 'highcharts';
+
+const merge = require('deepmerge');
+
+/**
+ * A hook to apply the Paste theme to Highcharts options.
+ *
+ * @param {Highcharts.Options} options - options to be provided into Highcharts for rendering.
+ * @returns {Highcharts.Options} optionsWithPasteTheme - Passed options with deepmerged Paste Theme
+ */
+export const usePasteHighchartsTheme = (options: Highcharts.Options): Highcharts.Options => {
+  const context = React.useContext(ThemeContext) as ThemeShape;
+  if (!context) {
+    throw new Error(
+      '[usePasteHighchartsTheme]: must be used within the @twilio-paste/theme provider. https://paste.twilio.design/introduction/for-engineers/manual-installation#setting-up-the-theme-provider'
+    );
+  }
+
+  return merge(options, {
+    colors: Object.values(context.dataVisualization),
```

Comment:
We originally only designed for these sequential color values, but I took the liberty of applying our text and font styles elsewhere to make the charts look more branded. The nice thing about this package is we can update the styles as needed and ship it as a patch, which folks can get for free.